src/mame/drivers/megaplay.cpp

```diff
@@ -329,65 +329,65 @@ static INPUT_PORTS_START ( mp_mazin )
 	PORT_INCLUDE( megaplay )
 
 	PORT_MODIFY("DSW1") // DSW C  (per game settings)
-	PORT_DIPNAME( 0x03, 0x02, "Initial Player" ) PORT_DIPLOCATION("SW3:1,2")
-	PORT_DIPSETTING( 0x00, "2" )
-	PORT_DIPSETTING( 0x01, "1" )
-	PORT_DIPSETTING( 0x02, "3" )
-	PORT_DIPSETTING( 0x03, "4" )
+	PORT_DIPNAME( 0x03, 0x01, "Initial Player" ) PORT_DIPLOCATION("SW3:1,2")
+	PORT_DIPSETTING( 0x02, "1" )
+	PORT_DIPSETTING( 0x03, "2" )
+	PORT_DIPSETTING( 0x01, "3" )
+	PORT_DIPSETTING( 0x00, "4" )
 	PORT_DIPNAME( 0x04, 0x04, DEF_STR ( Difficulty ) ) PORT_DIPLOCATION("SW3:3")
-	PORT_DIPSETTING( 0x04, DEF_STR ( Hard ) )
-	PORT_DIPSETTING( 0x00, DEF_STR ( Normal ) )
+	PORT_DIPSETTING( 0x04, DEF_STR ( Normal ) )
+	PORT_DIPSETTING( 0x00, DEF_STR ( Hard ) )
 	PORT_DIPNAME( 0x08, 0x08, "Title" ) PORT_DIPLOCATION("SW3:4")
 	PORT_DIPSETTING( 0x08, "EUROPE" )
 	PORT_DIPSETTING( 0x00, "U.S.A" )
 INPUT_PORTS_END
 
 static INPUT_PORTS_START ( mp_soni2 )
 	PORT_INCLUDE( megaplay )
-
+		
```

Comment:
Please remove this stray space so it doesn’t cause churn later.

Reply:
ran srcclean on it